client/js/netflix/watchScreen.js

```diff
@@ -0,0 +1,59 @@
+function createCastList(casts){
+    let castList=document.createElement('ol');
+    castList.className = "cast-list"
+    for(var i=0;i<casts.length;i++)
```

Comment:
can we use map method here? if not then at least forEach

Reply:
map for cast does not make sense. 
I will do the for each part, No worries.